make/mt-os/mt-stm32f4.mk

```diff
@@ -96,6 +101,7 @@ XI_COMPILER_FLAGS += -DHAVE_ECC
 XI_COMPILER_FLAGS += -DHAVE_TLS_EXTENSIONS
 XI_COMPILER_FLAGS += -DHAVE_AESGCM
 XI_COMPILER_FLAGS += -DALT_ECC_SIZE
+XI_COMPILER_FLAGS += -DSTM32F410xx
```

Comment:
is this required? if yes, then please remove line 46:
XI_COMPILER_FLAGS += -DSTM32F407xx
and use
XI_COMPILER_FLAGS += -DSTM32F417xx
since our eval board has this MCU on it.